### ISSUE TITLE
[FEATURE] Add support for corporate-hosted github

### ISSUE THREAD

**Issue by figgis** [apoclyps/reviews#119]
Currently the path to github is hardcoded.

Many corporations uses a hosted github-environment and it would be great of this tool would take that path as a config-parameter and use that if provided.

I got it to work using the following patch

```python
❯ git diff
diff --git a/reviews/source_control/client.py b/reviews/source_control/client.py
index ca94e5e..649d1a9 100644
--- a/reviews/source_control/client.py
+++ b/reviews/source_control/client.py
@@ -11,7 +11,7 @@ class GithubAPI:
     """Create and execute requests using the Github API"""

     def __init__(self) -> None:
-        self._client = Github(config.GITHUB_TOKEN, per_page=config.DEFAULT_PAGE_SIZE)
+        self._client = Github(config.GITHUB_TOKEN, per_page=config.DEFAULT_PAGE_SIZE, base_url="https://xxx-github.xxx.com/api/v3")
```

**Comment by apoclyps** [comment]
thanks, @figgis for opening this feature request. feel free to open a PR with the above change and I can review/approve it over lunch - alternatively, I can add it this evening (+8 hours from now).

**Comment by figgis** [comment]
Thanks for prompt reply. This is not in a proper state for a PR yet. New config value is needed and it needs to be propagated here etc. Not sure about the `api/v3` either. Just something I tried and it worked :)

**Comment by apoclyps** [comment]
@figgis I've implemented support for a configurable host to support Github Enterprise users in #121

If you have any other feedback or suggestions related to the above, I'd be happy to incorporate them into #121 otherwise I'll leave it open for a short time to allow others to review it before merging it into the next release (0.1.7).

**Comment by apoclyps** [comment]
closed by #121
